.travis.yml

```diff
@@ -6,7 +6,7 @@ install:
 
 script:
   # This line is required if using a branch of constellation:
-  # - pip3 install git+https://github.com/reside-ic/constellation@reside-57#egg=constellation
+  - pip3 install git+https://github.com/reside-ic/constellation@reside-61#egg=constellation
```

Comment:
can be reverted

Reply:
thanks - it's just rebuilding on travis now